packages/batch-submitter/src/exec/run-batch-submitter.ts

```diff
@@ -438,11 +438,15 @@ export const run = async () => {
       try {
         await func()
       } catch (err) {
-        logger.error('Error submitting batch', {
-          message: err.toString(),
-          stack: err.stack,
-          code: err.code,
-        })
+        logger.error(
+          // Tag with status or unknown error
+          `Error submitting batch: ${err.status ? err.status : 520}`,
+          {
+            message: err.toString(),
+            stack: err.stack,
+            code: err.code,
+          }
+        )
         logger.info('Retrying...')
```

Comment:
Do you happen to know what linter rule causes these characters to be placed on their own lines? Its very verbose. This isn't in the scope of your PR but its just something that sticks out to me